tests/Api/ScreenGroupsTest.php

```diff
@@ -172,7 +172,6 @@ public function testGetScreenGroupsScreenRelations(): void
             '@id' => '/v1/screen-groups',
             '@type' => 'hydra:Collection',
             'hydra:view' => [
-                '@id' => '/v1/screens/'.$ulid.'/screen-groups?itemsPerPage=2',
```

Comment:
Try changing it to `'@id' => '/v1/screens/'.$ulid.'/screen-groups?itemsPerPage=2&page=1',`